### ISSUE TITLE
[CI] Error on macos/opt build on main branch

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#8861]
```
[task 2023-12-22T07:05:01.485Z] Fetching tokens...
[task 2023-12-22T07:05:01.754Z] Outputting secret to: /opt/worker/tasks/task_169460466390766/checkouts/vcs/sentry_dsn
[task 2023-12-22T07:05:01.754Z] Traceback (most recent call last):
[task 2023-12-22T07:05:01.754Z]   File "/opt/worker/tasks/task_169460466390766/checkouts/vcs/./taskcluster/scripts/get-secret.py", line 106, in <module>
[task 2023-12-22T07:05:01.754Z]     main()
[task 2023-12-22T07:05:01.754Z]   File "/opt/worker/tasks/task_169460466390766/checkouts/vcs/./taskcluster/scripts/get-secret.py", line 94, in main
[task 2023-12-22T07:05:01.754Z]     write_secret_to_file(
[task 2023-12-22T07:05:01.754Z]   File "/opt/worker/tasks/task_169460466390766/checkouts/vcs/./taskcluster/scripts/get-secret.py", line 23, in write_secret_to_file
[task 2023-12-22T07:05:01.754Z]     value = data["secret"][key]
[task 2023-12-22T07:05:01.754Z] KeyError: 'sentry_dsn'
```

Marking this as high given it prevents QA from getting new MacOS builds.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-6054)
┆Reporter: Beatriz Rizental Machado

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

Points? 🙂

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

and fixVersion would be 2.20, right?

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Verified as fixed on macOS m1 - latest 2.20 (2.202401022228) builds are available in Task-cluster.

!image-20240103-125545.png|width=477,height=249!
